virttest/env_process.py

```diff
@@ -7,8 +7,11 @@
 import shutil
 import sys
 import copy
-import urllib2
 import multiprocessing
+try:
+    from urllib.request import ProxyHandler, build_opener, install_opener
```

Comment:
should be from six.moves.urllib.request?

Reply:
There are something unexpected error, when I try to use six

Reply:
ok, thanks